packages/caliper-ethereum/lib/ethereum-connector.js

```diff
@@ -402,8 +402,9 @@ class EthereumConnector extends ConnectorBase {
             if (txRcpt.status === '0x1') {
                 return new web3.eth.Contract(contractData.abi, txRcpt.contractAddress);
             } else {
-                logger.error('Failed private transaction hash ' + txHash);
-                throw new Error('Failed private transaction hash ' + txHash);
+                let msg = `Failed private transaction hash ${txHash}`;
```

Comment:
as msg never changes it should be `const` not `let`

Reply:
Oh, yes, sorry it happened by mistake. Updated it.